flink-contrib/flink-storm-compatibility/flink-storm-compatibility-examples/src/main/java/org/apache/flink/stormcompatibility/singlejoin/SingleJoinTopology.java

```diff
@@ -1,90 +0,0 @@
-/*
```

Comment:
If you prepare an PR, limit the changes to the feature you are implementing. This example is not related to the new functionality, thus, you should just leave it as it is ;)